args4j/examples/SampleMain.java

```diff
@@ -65,7 +68,7 @@ public void doMain(String[] args) throws IOException {
             // after parsing arguments, you should check
             // if enough arguments are given.
             if( arguments.isEmpty() )
-                throw new CmdLineException(parser,"No argument is given");
+                throw new CmdLineException(parser, Messages.ILLEGAL_PATH, "No argument is given");
```

Comment:
Hi @KarlFish ,

couldn't see the diffs in your Gist, this would help a lot when giving you feedback.
I see that you are not using `CmdLineException` anymore, this is good.

Merge the changes: You could simply commit & push the new version to your pull request branch in your fork, this will then add the revised code to the existing pull request in this project.

After that, @kohsuke will need to accept the patch. It looks like Kohsuke is quite busy at the moment.

Stephan